logging/logging.go

```diff
@@ -454,3 +475,33 @@ func createLogString(entry *sdjournal.JournalEntry, addUnit bool) (logStr string
 func getLogDate(entry *sdjournal.JournalEntry) (date time.Time) {
 	return time.Unix(int64(entry.RealtimeTimestamp/1000000), int64((entry.RealtimeTimestamp%1000000))*1000)
 }
+
+func getServiceNameFromLog(logEntry *sdjournal.JournalEntry) (serviceName string) {
+	serviceName = filepath.Base(logEntry.Fields[sdjournal.SD_JOURNAL_FIELD_SYSTEMD_CGROUP])
+	if !strings.Contains(serviceName, "aos_") {
+		// with cgroup v2 logs from container do not contains _SYSTEMD_UNIT due to restrictions
+		// that's why id should be checked via _SYSTEMD_CGROUP
+		// format: /system.slice/AOS_SERVICE_UUID
+
+		return fmt.Sprintf("aos_%s.service", serviceName)
+	}
+
+	return serviceName
+}
+
+func getServiceNameFilter(serviceName string) (filter string) {
```

Comment:
Why do you need function if it is used only once?

Reply:
just for readability. okay, removed